python-threatexchange/threatexchange/extensions/vpdq/tests/test_vpdq_faiss.py

```diff
@@ -23,7 +23,7 @@
         dedupe,
         quality_filter,
     )
-    from threatexchange.extensions.vpdq.tests.utils import (
+    from utils import (
```

Comment:
blocking: use `threatexchange.extensions.vpdq.tests.utils` instead of just `utils`

nit: I don't love that `threatexchange.tests.hashing.utils` exist as well

Reply:
Mypy doesn't like that because there isn't an init file in the test folder. So, mypy won't check that folder. Solution 1): type ignore here. 2): Add missing import 3) This in-place import 
I think the third one is better.